### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by michaelwolo**

### CODE REVIEW

**Comment by pg-infosec** [comment]
Vulnerable Package: eslint
Affected Range: < 4.18.2
Fixed Version: 4.18.2
Related CVE: WS-2018-0592
Severity: MODERATE

More Information: Please consider upgrading eslint to prevent deploying vulnerable code into production. For more information, visit: https://github.com/eslint/eslint/commit/f6901d0bcf6c918ac4e5c6c7c4bddeb2cb715c09, or visit the [FAQ](https://docs.google.com/document/d/1XqSqr-7oheQOVzofd2CLi1WWgOlSUw0SCttZPNI7DXQ)

**Comment by pg-infosec** [comment]
Vulnerable Package: webpack-dev-server
Affected Range: < 3.1.11
Fixed Version: 3.1.11
Related CVE: CVE-2018-14732
Severity: LOW

More Information: Please consider upgrading webpack-dev-server to prevent deploying vulnerable code into production. For more information, visit: https://nvd.nist.gov/vuln/detail/CVE-2018-14732, or visit the [FAQ](https://docs.google.com/document/d/1XqSqr-7oheQOVzofd2CLi1WWgOlSUw0SCttZPNI7DXQ)

File: src/components/fetch-wrapper.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Loading from './loading';
 
 function withFetching(WrappedComponent, props) {
   return class extends Component {
-    constructor(props) {
+    constructor(props) { // eslint-disable-line no-shadow
```
Comment (bohdan0) [inline src/components/fetch-wrapper.jsx]: curious what is this for?

Comment (michaelwolo) [reply src/components/fetch-wrapper.jsx]: I noticed this component has two variables called `props`, inside the params for `withFetching` and here in `constructor`. This was previously causing a lint error. I added this comment to minimally disrupt this logic.

**Pull request merged**